dev/com.ibm.ws.ssl/resources/com/ibm/ws/ssl/resources/ssl.nlsprops

```diff
@@ -653,3 +653,7 @@ ssl.client.handshake.cacerts.error.CWPKI0828E=CWPKI0828E: The trustDefaultCerts
 ssl.client.handshake.cacerts.error.CWPKI0828E.explanation=The default truststore is not properly configured to establish trust.  
 ssl.client.handshake.cacerts.error.CWPKI0828E.useraction=Review the extended error message to determine what to change in the configuration.
 # -------------------------------------------------------------------------------------------------
+ssl.update.certificate.error=CWPKI0829E: SSL certificate update error. {0}
+ssl.update.certificate.error.explanation=The SSL certificate could not be updated at the specified location.
+ssl.update.certificate.error.useraction=Ensure the location is accessible by the server process. Any FFDCs may indicate a fatal error in generating or loading the keys.
```

Comment:
`Any FFDCs may indicate a fatal error in generating or loading the keys.`

Does this mean that FFDC messages can be reviewed to learn more about the fatal error, or that the presence of FFDC messages in the log can be interpreted to mean there was a fatal error in load or capture? Wondering if the user action should be to review FFDCs or just look to see if there are any

Reply:
Updated